src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +54,17 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        Predicate<Person> personPredicate = new Predicate<Person>() {
+            @Override
+            public boolean test(Person person) {
+                return person.getAge() >= fromAge
+                        && person.getAge() <= toAge
+                        && person.getSex().equals(Person.Sex.MAN);
+            }
+        };
```

Comment:
```suggestion
        Predicate<Person> personPredicate =  person -> person.getAge() >= fromAge
                        && person.getAge() <= toAge
                        && person.getSex().equals(Person.Sex.MAN);
            }
        };
```
You could have wrote in this way